volatility3/framework/__init__.py

```diff
@@ -58,7 +58,7 @@ def __init__(self, value: Any, cls: Type) -> None:
         self.default_value = value
         self.cls = cls
 
-    def __get__(self, obj: Any, get_type: Type = None) -> Any:
+    def __get__(self, obj: Any, get_type: Type = Optional[None]) -> Any:
```

Comment:
This is the wrong place for the `Optional`?  It should be part of the type not the return value, shouldn't it?

Reply:
You're right, my bad 😅